ini/stm32f4.ini

```diff
@@ -46,6 +46,8 @@ extends             = stm32_variant
 board               = marlin_STM32F407ZGT6
 board_build.variant = MARLIN_FLY_F407ZG
 board_build.offset  = 0x8000
+build_flags         = ${stm32_variant.build_flags}
+                      -D_STM32_PLATDEFS=MARLIN_FLY_F407ZG
```

Comment:
It would be helpful if the platforms could provide the first analog pin number, or if we can acquire it from a macro or `constexpr` function somewhere, and then we don't need this extra level of convolution.

Reply:
Yes, I agree! But they don't. The implementation should be entirely rewritten and not assume that analog pins are within an incremental range. Please look into PinMap_ADC from the Arduino Core and look up the pins dynamically. That is my improvement idea for somebody who wants to create the Definitive Edition PINS_DEBUGGING feature PR.